Microsoft.Dynamics365.UIAutomation.Browser/Extensions/TimeExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Created by: Rodriguez Mustelier Angel (rodang)
+// Modify On: 2020-02-09 14:05
+
+using System;
+
+namespace Microsoft.Dynamics365.UIAutomation.Browser
+{
+    public static class TimeExtensions
+    {
+        public static TimeSpan Seconds(this int value)
+            => TimeSpan.FromSeconds(value);
+    }
+}
```

Comment:
sintaxis sugar: make the code more expressive. instead new TimeSpan(0, 0, 60) or TimeSpan.FromSeconds(60) => 60.Seconds(), => ex: Wait(60.Seconds())  or ThinkTime(60.Seconds())